server/lib/src/repl/consumer.rs

```diff
@@ -237,14 +204,16 @@ impl<'a> QueryServerWriteTransaction<'a> {
     pub fn consumer_apply_changes(
         &mut self,
         ctx: &ReplIncrementalContext,
-    ) -> Result<(), OperationError> {
+    ) -> Result<ConsumerState, OperationError> {
         match ctx {
             ReplIncrementalContext::NoChangesAvailable => {
                 info!("no changes are available");
-                Ok(())
+                Ok(ConsumerState::Ok)
             }
             ReplIncrementalContext::RefreshRequired => {
-                todo!();
+                error!("Unable to proceed with consumer incremental - the supplier has indicated that our RUV is outdated, and replication would introduce data corruption.");
+                error!("This servers content must be refreshed to proceed. If you have nominated for automatic refresh, this will occur shortly.");
```

Comment:
```suggestion
                error!("This server's content must be refreshed to proceed. If you have configured automatic refresh, this will occur shortly.");
```